homeassistant/components/meteo_france/weather.py

```diff
@@ -83,7 +84,9 @@ def attribution(self):
     @property
     def forecast(self):
         """Return the forecast."""
-        reftime = datetime.now().replace(hour=12, minute=00)
+        reftime = dt_util.now(dt_util.get_time_zone("Europe/Paris")).replace(
```

Comment:
The timestamp that we report should be utc iso formatted datetime.

Reply:
OK conversion to UTC added.